docs/maintenance/Pull_Requests.mdx

```diff
@@ -111,6 +108,21 @@ For preliminary reviews, be clear with what scale you're reviewing at: _"Reviewi
 For repeat reviews, be clear when it's something you missed earlier and/or there's new information.
 Don't apologize if the missed information was only made clear because of requested changes - this is part of the review process!
 
+### Approvals
+
+If the PR addresses a time critical task, such as a security fix or `main` branch build break, go ahead and squash merge it.
+
+Otherwise, upon completing your review, if the build is passing and you have no blockers, approve it on GitHub.
+Then:
+
+- If there isn't a `1 approval` label or existing approval, add the `1 approval` label
+- If there's already `1 approval` and/or it's been a week since the last request for changes, go ahead and squash merge
+  - For straightforward PRs that don't impact users, you can wait 3 days instead
+
+There's no need to reset waiting periods for minor fixups from code reviews of otherwise approved code.
+
+We try to leave PRs open for at least a week to give reviewers who aren't active every day a chance to get to it.
```

Comment:
Made this a bit more clear/precise: it's a week in review, not a week since approval. IMO no need to wait a week after approval for something that's already been open for a bit.

Reply:
Oh, good call on clarifying that. That's a better approach than what my understanding was! 👍